trusted-firmware-m/platform/ext/target/nordic_nrf/nrf5340dk_nrf5340_cpuapp/RTE_Device.h

```diff
@@ -40,13 +40,13 @@
 #define   RTE_USART1                    1
 //   <h> Pin Selection (0xFFFFFFFF means Disconnected)
 //     <o> TXD
-#define   RTE_USART1_TXD_PIN            25
+#define   RTE_USART1_TXD_PIN            33
 //     <o> RXD
-#define   RTE_USART1_RXD_PIN            26
+#define   RTE_USART1_RXD_PIN            32
 //     <o> RTS
-#define   RTE_USART1_RTS_PIN            0xFFFFFFFF
+#define   RTE_USART1_RTS_PIN            11
 //     <o> CTS
-#define   RTE_USART1_CTS_PIN            0xFFFFFFFF
+#define   RTE_USART1_CTS_PIN            10
```

Comment:
These pins are [assigned to UART in the nrf5340dk_nrf5340_cpunet board](https://github.com/zephyrproject-rtos/zephyr/blob/1d4c36486c94212478bf222e12bf463affb77d55/boards/arm/nrf5340dk_nrf5340/nrf5340dk_nrf5340_cpunet.dts#L93-L96), so I'm not convinced we should use them for the secure UART in TF-M.

Reply:
I see that for the PDK we already got such conflict: pins P0.25 and P0.26 are used by the secure UART https://github.com/zephyrproject-rtos/trusted-firmware-m/blob/74f148fee35d7019f5b74ebb562f78cf6f988804/trusted-firmware-m/platform/ext/target/nordic_nrf/nrf5340pdk_nrf5340_cpuapp/RTE_Device.h#L42-L45
and by the [UART in the nrf5340pdk_nrf5340_cpunet board](https://github.com/zephyrproject-rtos/zephyr/blob/1d4c36486c94212478bf222e12bf463affb77d55/boards/arm/nrf5340dk_nrf5340/nrf5340pdk_nrf5340_cpunet.dts#L93-L94) in Zephyr. I guess that should be fixed.

Reply:
What happens if both cores use the same UART pins?

Reply:
> What happens if both cores use the same UART pins?

Only the core to which those pins are assigned (via `PIN_CNF[n].MCU_SEL`) will be able to access them.